.htaccess

```diff
@@ -77,8 +77,7 @@ deny from env=stayout
 	RewriteRule Capfile - [F]
 	RewriteRule Gemfile - [F]
 	RewriteRule Gemfile\.lock - [F]
-	RewriteRule Gruntfile\.coffee - [F]
-	RewriteRule Sumofile - [F]
+	RewriteRule gulpfile\.js - [F]
```

Comment:
Actually, gulpfile.js, package.json & yarn.lock should be added in the Fork forbidden folders as they also exist in forkcms/forkcms